integration-tests/load/automationv2_1/automationv2_1_test.go

```diff
@@ -563,6 +528,8 @@ Load Config:
 		l.Error().Err(err).Msg("Error sending slack notification")
 	}
 
+	// a.ChainClient.Cfg.PendingNonceProtectionEnabled = false
```

Comment:
@Tofel should this be removed if not needed? Also, what exactly would it do if disabled?